src/cascadia/TerminalApp/TerminalTab.h

```diff
@@ -7,6 +7,9 @@
 #include "TabBase.h"
 #include "TerminalTab.g.h"
 
+static constexpr uint32_t HeaderRenameBoxWidthDefault{ 165 };

```

Comment:
To be clear - this 165 is just _coincidentally_ the size of the tab, right? if we were to ever allow the user to change the size of the tabs (by setting a static size, not like SizeToContent), then we'd need to update this value to, right?

I'm not saying we need to have a plan now, I just want to make sure we we know what that magic number means

Reply:
Yes, its the size so that the rename box doesn't clip with the tab view item's edges